vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1006,7 +1010,7 @@ def inplace_fused_experts(hidden_states: torch.Tensor,
                           use_int8_w8a8: bool = False,
                           use_int8_w8a16: bool = False,
                           use_int4_w4a16: bool = False,
-                          use_mxfp4_w4a4: bool = False,
+                          ocp_mx_scheme: Optional[str] = None,
```

Comment:
Nit: `Optional[OCP_MX_Scheme]`? It's not a huge deal but, imo, using the enum is slightly better. Same for other spots where you use `Optional[str]`